src/_P039_Thermosensors.ino

```diff
@@ -61,9 +61,9 @@
 // #include <String.h>
 
 // // plugin-local quick activation of debug messages
-#ifdef BUILD_NO_DEBUG
-  #undef BUILD_NO_DEBUG
-#endif
+// #ifdef BUILD_NO_DEBUG
+//   #undef BUILD_NO_DEBUG
+// #endif
```

Comment:
Good one to remove it here...

Reply:
I missed it in the last push - might be the reason for some of the build fails. Another candidate could be the missing uint32_t casts below...

Reply:
Solved in upcoming push !